common/walker/src/report.rs

```diff
@@ -123,9 +126,97 @@ impl SplitScannerError for Result<Report, ScannerError> {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, serde::Deserialize, serde::Serialize)]
+pub enum ReportType {
+    SBOM(String),
+    VEX(String),
+}
 /// Handle the report
-pub async fn handle_report(report: Report) -> anyhow::Result<()> {
-    // FIXME: this is a very simplistic version of handling the error
-    log::info!("Import report: {report:#?}");
+pub async fn handle_report(report: Report, report_type: ReportType) -> anyhow::Result<()> {
```

Comment:
I don't like the error handling in this function.

In some cases it ignores the error. In some cases it logs the error. In some cases it propagates the error.

There should be a clear strategy of what makes the function fail, and what is acceptable (logged or ignored).